### PR TITLE
Added Game Over Modal with logic

### PR DESCRIPTION
**Pull request by mgoniszewska**

### CODE REVIEW

File: src/layouts/GameOverModal/styles.css [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+  border-radius: 16px;
+  box-shadow: 0 0 20px rgba(255, 0, 0, 0.9);
+}
+
+@media only screen and (min-width: 360px) {
+}
+
+@media only screen and (min-width: 768px) {
+  .modal-container__game-over-modal {
+    position: absolute;
```
Comment (FilipHalon) [inline src/layouts/GameOverModal/styles.css:114]: You don't have to redeclare styles if they are the same as above. Styles concerning an element that are lowest in a file will be the ones a browser uses and this includes media queries.

**Review by FilipHalon — CHANGES_REQUESTED** [review]

File: src/layouts/GameOverModal/GameOverModal.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import './styles.css';
+import Button from '../../components/button/Button';
```
[the remark this answers is not in the document]
Comment (mgoniszewska) [reply src/layouts/GameOverModal/GameOverModal.js:2]: I noticed that we don't use absolute imports at all. How can we do it in this project? I read on stackoverflow that we should have configuration in vite for it. Maybe you can configure it and then we will update our imports.

[the remark this answers is not in the document]
Comment (FilipHalon) [reply src/layouts/GameOverModal/GameOverModal.js:2]: Right. As we discussed, let's leave it relative as long as we don't configure Vite for it.

File: src/layouts/GameOverModal/GameOverModal.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    inputContainer.className = 'modal-container__input-container';
+    modalInput.className = 'modal-container__input';
+    modalInputDescription.className = 'modal-container__input-description';
+    modalButtonContainer.className = 'modal-container__button';
+    closeModalButtonContainer.className = 'modal-container__close-button-container';
+
+    modalHeader.innerText = 'game over';
+    modalSummary.innerText = `The force is strong in you, Padawan! During 1 minute you have answered ${this.playerAnsweredCount} questions. And computer quessed ${this.pcAnsweredCount}.`;
+    modalInputDescription.innerText = 'Please, fill your name in order to receive eternal glory in whole Galaxy!';
+    yodaImage.src = '../../images/Yoda.png';
```
[the remark this answers is not in the document]
Comment (FilipHalon) [reply src/layouts/GameOverModal/GameOverModal.js:76]: This still could be shortened to `'/images/Yoda.png'`.

**Review by FilipHalon — APPROVED** [review]

**Comment by ike-tom** [comment]
Modal is not centered in the middle of the page on mobiles

**Review by ike-tom — CHANGES_REQUESTED** [review]
One comment from github+ I wrote you more on discord

**Review by ike-tom — APPROVED** [review]

**Pull request merged**